fonts/OTF/TeX/makeFF

```diff
@@ -1517,11 +1517,12 @@ $map{cmtt10} = {
     0x15 => [0x306,-525,0], # \breve (combining)
     0x16 => [0x304,-525,0], # \bar (combining)
     0x17 => [0x30A,-525,0], # ring above (combining)
+    0x20 => 0x2423,         # graphic representation of space
```

Comment:
But the glyph at position 0x20 is a `␣` not a space. I guess LaTeX doesn't actually use space characters, because spaces are usually strecthable nonetheless, so it will insert horizontal positioning instructions into the PostScript (and later PDF) and doesn't need a character for this.